brainscore_vision/benchmarks/hebart2023/benchmark.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+
+from brainscore_vision import load_dataset
+from brainscore_vision.benchmarks import BenchmarkBase
+from brainscore_vision.benchmark_helpers.screen import place_on_screen
+from brainscore_vision.model_interface import BrainModel, BehavioralAssembly
+from brainscore_vision.metrics import Score
+
+BIBTEX = """@article{10.7554/eLife.82580,
+          author = {Hebart, Martin N and Contier, Oliver and Teichmann, Lina and Rockter, Adam H and Zheng, Charles Y and Kidder, Alexis and Corriveau, Anna and Vaziri-Pashkam, Maryam and Baker, Chris I},
+          journal = {eLife},
+          month = {feb},
+          pages = {e82580},
+          title = {THINGS-data, a multimodal collection of large-scale datasets for investigating object representations in human brain and behavior},
+          volume = 12,
+          year = 2023
+          }"""
+
+class Hebart2023Accuracy(BenchmarkBase):
+    def __init__(self, similarity_measure='dot'):
+        self._similarity_measure = similarity_measure
+        self._visual_degrees = 8
+        self._number_of_trials = 1
+        self._assembly = load_dataset('Hebart2023')
+
+        print(self._assembly.stimulus_set)
+
+        super().__init__(
+            identifier=f'Hebart2023Accuracy_{similarity_measure}', version=1,
+            ceiling_func=lambda: Score([0.6844, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation']),
+            parent='Hebart2023',
+            bibtex=BIBTEX
+        )
+
+    def __call__(self, candidate: BrainModel):
+        triplets = np.array([
+            self._assembly.coords["image_1"].values,
+            self._assembly.coords["image_2"].values,
+            self._assembly.coords["image_3"].values
+        ]).T.reshape(-1, 1)
+
+        triplets = BehavioralAssembly(triplets, coords={None}, dims=['presentation'])
```

Comment:
How does this look? Right now those are just the integer values corresponding to the images. But in the interface we have ``'image1.png'`` etc. 

Doing ``triplets = [f"{triplet}.png" for triplet in triplets]`` before creating the assembly should fix this right?